Tasks/QuickPerfTest/task.json

```diff
@@ -99,6 +99,17 @@
         "Australia East": "Australia East (New South Wales)",
         "Australia Southeast": "Australia Southeast (Victoria)"
       }
+    },
+    {
+      "name": "machineType",
+      "type": "radio",
+      "label": "Run load test using",
+      "required": true,
+      "defaultValue": "0",
+      "options": {
+        "2": "Self-provisioned agents",
```

Comment:
Just see if you want to keep the order uniform in all tasks even though if it doesn't matter.